build/webpack/webpack.extension.web.config.js

```diff
@@ -89,6 +90,7 @@ const config = {
     },
     externals: ['vscode', 'commonjs', 'electron'], // Don't bundle these
     plugins: [
+        ...common.getDefaultPlugins('web'),
```

Comment:
So we can analyze the bundle using `bundle-analyzer`, this change doesn't impact the build only generates an html file during dev build based on an env variable.